### PR TITLE
Updated converter for popular articles

### LINKED ISSUE #157: [Blog/News] Implement popular articles (static)
Implement popular articles used in blog & news sidebar in a first static version. See screenshot.

- [ ] render article list based on links passed via the block, data should be loaded from blog index
- [ ] render view all link

Screenshot 2023-09-29 at 13 23 34

### PR DESCRIPTION
**Pull request by rgravitvl**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #157 
Test URLs:
- Before: https://main--danaher-ls-aem--hlxsites.hlx.page/
- After: https://157-article-converter-update--danaher-ls-aem--hlxsites.hlx.page/

### CODE REVIEW

File: tools/actions/convert/test/fixtures/blog-semantic.html [hunk 1]
```diff
@@ -141,13 +142,35 @@ <h2>9. Skyland PIMS</h2>
         </div>
       </div>
       <p>In conclusion, these nine products are revolutionizing <a href="/content/danaher/ls/us/en/solutions/mrna-therapy/mrna-development-manufacturing.html" title="mRNA Development and Manufacturing"><strong>mRNA development and manufacturing</strong></a>. By integrating these tools into your workflow, you can accelerate your research, improve your outcomes, and stay at the forefront of life sciences innovation. Explore these products and request a quote today to see how they can transform your lab operations.</p>
+      <div class="article popular">
```
Comment (mhaack) [inline tools/actions/convert/test/fixtures/blog-semantic.html:145]: I suggest to name the block `Related Articles (popular)`

File: tools/actions/convert/test/fixtures/blog-semantic.html [hunk 2]
```diff
@@ -141,13 +142,35 @@ <h2>9. Skyland PIMS</h2>
...
+      </div>
+      <hr>
       <div class="section-metadata">
         <div>
           <div>style</div>
           <div>bg-danaherlightblue-50</div>
         </div>
       </div>
       <hr>
+      <div class="article recent">
```
Comment (mhaack) [inline tools/actions/convert/test/fixtures/blog-semantic.html:162]: I suggest to name the block `Related Articles (recent)`

**Review by mhaack — CHANGES_REQUESTED** [review]

File: tools/actions/convert/test/fixtures/blog-semantic.html (hunk 1 above)
Comment (rgravitvl) [reply tools/actions/convert/test/fixtures/blog-semantic.html]: Done

File: tools/actions/convert/test/fixtures/blog-semantic.html (hunk 2 above)
Comment (rgravitvl) [reply tools/actions/convert/test/fixtures/blog-semantic.html]: Done

**Review by mhaack — APPROVED** [review]

**Pull request merged**